ideascube/conf/base.py

```diff
@@ -302,3 +302,6 @@
 
 SESSION_COOKIE_AGE = 60 * 60  # Members must be logged out after one hour
 BACKUP_FORMAT = 'zip'  # One of 'zip', 'tar', 'bztar', 'gztar'
+
+CATALOG_CACHE_BASE_DIR = '/var/cache/ideascube/catalog'
```

Comment:
No need to reflect this in Debian package files?
IIRC, putting a path in Debian `dirs` will create it automatically (we could do so in python too), and in particular will ask the user what to do with the folder content when deinstalling the package.
But you are the packaging expert, not me, so I may have missed something :)